Himanshu_Kumar/project_Profile/index.html

```diff
@@ -0,0 +1,147 @@
+            est. Cras mist ipsum, consectetur ipsum in, egestas tellus ipsu vestibulum tellus accumsan nec tristique justo.
+          </p>
+
+        </div>
+        <div class="col-lg-4">
+          
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-lg-8">
+          <h1>sfdghjxgfchgjvfgghvh</h1>
+          <h2>324567354567845678567</h2>
+          <p>!@#$%^&#$%^&*$%^&*(
+            Nulla pellentesque mi non laoreet eleifend. Integer sed porttitor mollisar lorem, at molestie arcust pulvinar ut. Proina fermentum 
+            est. Cras mist ipsum, consectetur ipsum in, egestas tellus ipsu vestibulum tellus accumsan nec tristique justo.
+            Nulla pellentesque mi non laoreet eleifend. Integer sed porttitor mollisar lorem, at molestie arcust pulvinar ut. Proina fermentum 
+            est. Cras mist ipsum, consectetur ipsum in, egestas tellus ipsu vestibulum tellus accumsan nec tristique justo.
+            Nulla pellentesque mi non laoreet eleifend. Integer sed porttitor mollisar lorem, at molestie arcust pulvinar ut. Proina fermentum 
+            est. Cras mist ipsum, consectetur ipsum in, egestas tellus ipsu vestibulum tellus accumsan nec tristique justo.
+            Nulla pellentesque mi non laoreet eleifend. Integer sed porttitor mollisar lorem, at molestie arcust pulvinar ut. Proina fermentum 
+            est. Cras mist ipsum, consectetur ipsum in, egestas tellus ipsu vestibulum tellus accumsan nec tristique justo.
+          </p>
+
+        </div>
+        <div class="col-lg-4">
+          
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-lg-8">
+          <h1>sfdghjxgfchgjvfgghvh</h1>
+          <h2>324567354567845678567</h2>
+          <p>!@#$%^&#$%^&*$%^&*(
+            Nulla pellentesque mi non laoreet eleifend. Integer sed porttitor mollisar lorem, at molestie arcust pulvinar ut. Proina fermentum 
+            est. Cras mist ipsum, consectetur ipsum in, egestas tellus ipsu vestibulum tellus accumsan nec tristique justo.
+            Nulla pellentesque mi non laoreet eleifend. Integer sed porttitor mollisar lorem, at molestie arcust pulvinar ut. Proina fermentum 
+            est. Cras mist ipsum, consectetur ipsum in, egestas tellus ipsu vestibulum tellus accumsan nec tristique justo.
+            Nulla pellentesque mi non laoreet eleifend. Integer sed porttitor mollisar lorem, at molestie arcust pulvinar ut. Proina fermentum 
+            est. Cras mist ipsum, consectetur ipsum in, egestas tellus ipsu vestibulum tellus accumsan nec tristique justo.
+            Nulla pellentesque mi non laoreet eleifend. Integer sed porttitor mollisar lorem, at molestie arcust pulvinar ut. Proina fermentum 
+            est. Cras mist ipsum, consectetur ipsum in, egestas tellus ipsu vestibulum tellus accumsan nec tristique justo.
+          </p>
+
+        </div>
+        <div class="col-lg-4">
```

Comment:
what's the purpose of empty div here ?

Reply:
Since using col-lg, I have kept it to add something in future like images,Projects, etc.

Reply:
keeping space for future work.... Still to add thhings